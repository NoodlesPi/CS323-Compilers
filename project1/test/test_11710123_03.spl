int func_1(int a)
{
  return 3a;
}
int func_2(int x, y)
{
  return (x + y);
}