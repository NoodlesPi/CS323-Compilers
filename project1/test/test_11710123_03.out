Error type A at Line 3: unknown lexeme 3a
Error type B at Line 5: Missing specifier 
